# Crcl contains maximum motion rates for canonical robot command language exchange.

float64   maxvel # maximum vel for each motion
float64   maxacc # maximum acc for each motion
float64   maxjerk # maximum jerk for each motion



